deployment/ansible/roles.yml

```
@@ -0,0 +1,8 @@
+- src: azavea.pip
+  version: 2.0.0
+
+- src: azavea.docker
+  version: 5.0.0
```

This version bump may require changes to the `group_vars`, but I think we should do it.

```suggestion
  version: 6.0.0
```

See: https://github.com/azavea/ansible-docker/blob/develop/CHANGELOG.md#600

---

We currently have these in `group_vars`:

```
docker_version: "5:19.*"
docker_compose_version: "1.26.*"
```

What should they be upgraded to?

---

I took a closer look at the changes, and I don't think you need to update anything. I'd verify things work after the bump.

---

They do indeed! Thanks for checking.